WORKSPACE

```diff
@@ -203,3 +203,21 @@ nodejs_register_toolchains(
 load("@bazel_skylib//:workspace.bzl", "bazel_skylib_workspace")
 
 bazel_skylib_workspace()
+
+load("@bazel_tools//tools/build_defs/repo:git.bzl", "git_repository")
+
+git_repository(
+      name = "io_bazel_rules_dotnet",
+      remote = "https://github.com/bazelbuild/rules_dotnet",
+      branch = "master",
```

Comment:
Nit: run `buildifier` to format `WORKSPACE`, ideally when you save a file so you don't have to think about it. 4 space indent.